### DISCUSSION TITLE
Add all app dependencies to APP_SHARED_DEPS

### DISCUSSION THREAD

**Discussion by dmtkpv** [directus/directus#12043] [Feature Requests]
Currently `APP_SHARED_DEPS` is
```js
export const APP_SHARED_DEPS = ['@directus/extensions-sdk', 'vue', 'vue-router', 'vue-i18n', 'pinia'];
```
There is no `axios `, `lodash`, `vuedraggable`, etc
Maybe it's possible to read app `package.json` and add its dependencies to `APP_SHARED_DEPS`?

**Comment by rijkvanzanten** [comment]
I believe the list in here is bundled differently in the app, which makes it slightly less optimized which in turn means we wouldn't want to share _everything_, but I might be mistaken. @nickrum could you elaborate?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
